packages/atlas/src/hooks/useCountdown.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react'
+
+import Timeout = NodeJS.Timeout
+
+export const useCountdown = (seconds?: number) => {
```

Comment:
Where is this used?

Reply:
[useCountdown](https://github.com/Joystream/atlas/pull/2218/files#diff-452a81aec0cd71561d35a7d0c694335b700ad5cf020154dacde2f45fca339e40R79)

We talked some time ago about creating 1 hook for counting down

Reply:
This has been already done here: #2204. This hook will provide you current time and will update it every second. So using this and target date, you can just find time left in the component

Reply:
Sorry, haven't noticed